### ISSUE TITLE
Gateway Drug Commitment Device AKA Mega-Commitwall

### ISSUE THREAD

**Issue by dreeves** [beeminder/blog#256]
I think it's worth announcing this on the blog, maybe after a couple days if we have a sense that we're not going to immediately undo it.

```[tasklist]
### Desiderata
- [x] Preview link
- [x] Keywords/tags
- [x] Image
- [x] Publish
- [x] Blog blurbs
- [x] Tweet: https://twitter.com/bmndr/status/1634292785565294592
- [x] NIX: Tip of the day
```

## Cognata

* https://forum.beeminder.com/t/gateway-drug-meta-commitment-device/10848

_**Verbata**: feature announcements, soup-naziing signups,_

**Comment by dreeves** [comment]
new mega-commitwall: no signing up for beeminder without hard-committing to create a goal

I ran this by you all a couple weeks ago under the heading of "soup-naziing signups" and, well, it's not the most uncontroversial thing we've tried but mostly no one thought it was too crazy of an idea so, as of just now, we're trying it!

Of course it doesn't matter for you veterans but if you're curious or want to give us feedback, open an incognito window and head to https://www.beeminder.com/users/sign_up and imagine how terrified you'd have been as a new newbee seeing that.

(If it goes well we'll probably be tempted to bump up the amount at risk to something like $20 instead of the current $5. Or jump to the logical conclusion of all this and require the creation of a meta goal as part of signup. But getting that right will take a lot more thought than this simple gateway drug commitment device did.)
